src/views/RecIM/components/Forms/SeriesForm/index.jsx

```diff
@@ -43,103 +58,61 @@ const SeriesForm = ({
-      required: true,
-    },
-    {
-      label: 'Series End Date',
-      name: 'endDate',
-      type: 'datetime',
-      error: errorStatus.endDate,
-      helperText: '*Required',
-      required: true,
-    },
-  ];
-
-  if (series) {
-    createSeriesFields.push(
-      {
-        label: 'Series Status',
-        name: 'statusID',
-        type: 'select',
-        menuItems: statuses.map((status) => {
-          return status.Description;
-        }),
-        error: errorStatus.statusID,
-        helperText: '*Required',
-        required: true,
-      },
-      {
-        label: 'Teams',
-        name: 'TeamIDs',
-        type: 'multiselect',
-        menuItems: activityTeams.map((team) => {
-          return team.Name;
-        }),
-        error: errorStatus.TeamIDs,
-        helperText: '*Required',
-        required: true,
-      },
-    );
-  } else {
-    createSeriesFields.push(
-      {
-        label: 'Series Type',
-        name: 'typeID',
-        type: 'select',
-        menuItems: seriesType.map((seriesType) => {
-          return seriesType.Description;
-        }),
-        error: errorStatus.type,
-        helperText: '*Required',
-        required: true,
-      },
-      {
-        label: 'Reference Series',
-        name: 'referenceSeriesID',
-        type: 'select',
-        menuItems: existingActivitySeries.map((series) => {
-          return series.Name;
-        }),
-        helperText: '*Optional',
-        required: false,
-      },
-      {
-        label: 'Number of Teams',
-        name: 'numberOfTeamsAdmitted',
-        type: 'text',
-        error: errorStatus.numberOfTeamsAdmitted,
-        helperText: '*Invalid Number',
-        required: true,
-      },
-    );
-  }
+  const additionalFields = series
+    ? [
+        {
+          label: 'Series Status',
+          name: 'statusID',
+          type: 'select',
+          menuItems: statuses.map((status) => status.Description),
+          helperText: '*Required',
+          required: true,
+        },
+        {
+          label: 'Teams',
+          name: 'TeamIDs',
+          type: 'multiselect',
+          menuItems: activityTeams.map((team) => team.Name),
+          helperText: '*Required',
+          required: true,
+        },
+      ]
+    : [
+        {
+          label: 'Series Type',
+          name: 'typeID',
+          type: 'select',
+          menuItems: seriesType.map((seriesType) => seriesType.Description),
+          helperText: '*Required',
+          required: true,
+        },
+        {
+          label: 'Reference Series',
+          name: 'referenceSeriesID',
+          type: 'select',
+          menuItems: existingActivitySeries.map((series) => series.Name),
+          helperText: '*Optional',
+          required: false,
+        },
+        {
+          label: 'Number of Teams',
+          name: 'numberOfTeamsAdmitted',
+          type: 'number',
+          min: 1,
+          helperText: '*Invalid Number',
+          required: true,
```

Comment:
Per Amos, this should not be required, we can allow empty
```suggestion
          min: 0,
          helperText: '*Invalid Number',
```